contrib/packs/actions/workflows/deploy.yaml

```diff
@@ -26,7 +26,23 @@ workflows:
 
         on-complete:
           - do_auto_install: <% $.auto_deploy = true %>
-          - do_manual_install: <% $.auto_deploy = false %>
+          - check_for_no_subtree: <% $.auto_deploy = false %>
+
+      check_for_no_subtree:
+        action: core.noop
+
+        on-complete:
+          - do_manual_install: <% len($.packs) != 0 %>
+          - publish_packs: <% len($.packs) = 0 %>
+
+      publish_packs:
+        action: core.noop
+
+        publish:
+          packs: [ <% $.repo_name %> ]
```

Comment:
It's because if where is no sub tree in the repo the repo_name and pack are mostly likely to have the same name. With this check and publish in place when calling the install from ChatOps you do not need to supply the same text as repo_name and packs.

So before the ChatOps alias was:

`Pack deploy my-pack packs my-pack branch dev`

Is now now instead:

`Pack deploy my-pack branch dev`

In the case of st2contrib packs will always have at least a single pack to install, so publish_packs will never be reached. As this only happens if packs has a 0 length.